### ISSUE TITLE
Task headers

### ISSUE THREAD

**Issue by StevenClontz** [rbeezer/mathbook#1548]
In HTML builds there's some CSS styling that gets `<h6>(a)</h6>` inline with the following paragraph:

![image](https://user-images.githubusercontent.com/1559632/126666929-f386daba-5331-46ba-9499-493770fd5a8e.png)

We should replicate this in slides:

![image](https://user-images.githubusercontent.com/1559632/126666860-eae9407d-532c-44c8-9eff-30837c3ba6a0.png)

While we're mucking around here, I'd like to propose that task headers be written as `(Task a)` rather than just `(a)`, to better distinguish from `<ol label="a"/>`.

![image](https://user-images.githubusercontent.com/1559632/126667176-3dfc1180-e46d-452c-ad38-0a4989aab7cc.png)

Proposed change (or maybe '(Part a)'?):

![image](https://user-images.githubusercontent.com/1559632/126667291-71acf609-43ea-4500-a981-582c9ed74972.png)

**Comment by StevenClontz** [comment]
The list within the task demonstrated is not a sequence of subtasks - it's enumerating multiple choices for students to consider during the task. I'm only slightly insulted that you'd suspect me of expressing tasks as an ordered list in PreTeXt. :-)

I wonder if the repeated "Task" is perhaps no less visual clutter than a repeated "Activity" or "Theorem".

The examples to compare are

- https://stevenclontz.github.io/cal2-tbil-2021/slides.html#/2/1 
- https://stevenclontz.github.io/cal2-tbil-2021/TI1.html#activity-1
 
But things are under rapid development at a workshop I'm helping run this week, so here's some commit sources in case those get broken: 

- https://github.com/StevenClontz/cal2-tbil-2021/blob/770b7f2bb154b07d2dd0ad02b4bb931d6b66db24/source/01-TI/01.ptx#L12
- https://github.com/StevenClontz/cal2-tbil-2021/blob/770b7f2bb154b07d2dd0ad02b4bb931d6b66db24/docs/slides.html#L48
- https://github.com/StevenClontz/cal2-tbil-2021/blob/770b7f2bb154b07d2dd0ad02b4bb931d6b66db24/docs/TI1.html#L104

**Comment by Alex-Jordan** [comment]
My opinion: labeling these with lowercase alph is right, as they come in a sequence right under something that is numbered with arabic. There are places where I find the styling on "(a)" too heavy, but that is something I have been meaning to report separately.

Is it more traditional for multiple choice options to use uppercase Alph? 

Thinking out loud, multiple choice options are a special thing. Not really an ordered list, except maybe sometimes there are some options you want to appear last. And not an unordered list, because each item needs a unique label. And not a description list because there is rarely a good option for a title for each item.

**Comment by StevenClontz** [comment]
There are pedagogical reasons to avoid permuting within a multiple choice list, particularly when the MCQ is being used for instruction rather than assessment, as they would be in an `<activity/>`.

Perhaps the lowercase `(a)` for task and `A.` for MCQ choices is a sufficient distinction.

**Comment by Alex-Jordan** [comment]
> There are pedagogical reasons to avoid permuting within a multiple choice list,

Right, and to clarify, the possibility to permute is not what I mean by suggesting that multiple choice options are not an "ol". What I mean is, there is no intrinsic complete ordering to the options. Each item needs a distinct identifier, for sure. And it's tradition to use ordered enumeration for those identifiers. But they might as well be ★, ●, ⚇, ▱, etc.

There is often a partial ordering to the items, where something like "none of the above" should come at the end. WeBWorK handling of multiple choice sets respects this (to a degree) with input syntax like `[[blue, black, gray],teal]`. In practice the ordering of `[blue, black, gray]` is randomized, and all three are presented before `teal`. But again I don't mean to focus on the random ordering.

Please consider this open philosophizing about the nature of multiple choice, not (yet) a proposal to change how PreTeXt handles multiple choice options.

**Comment by StevenClontz** [comment]
Here's an example that caused communication tripups during a mock teaching session this afternoon with the part-letters and choice-letters.

![image](https://user-images.githubusercontent.com/1559632/126697132-0df04a81-ae76-44ad-befc-712ca86a8b7d.png)

Since choice-letters are frequently alphabetical, it makes me wish tasks were numerical for ease of distinction. (I suspect maybe this specific question could be posed better, but I think the general case is still problematic.)

> Please consider this open philosophizing about the nature of multiple choice, not (yet) a proposal to change how PreTeXt handles multiple choice options.

Noted, although I think what this may be revealing is that maybe there's a need for `<choices/>` distinct from `<ol/>` for project-likes. Authors probably shouldn't care about the labeling of their multiple choice, and PreTeXt maybe should ensure that choices are labeled in a way that doesn't conflict with how e.g. tasks are labeled.
